MediaBrowser.Api/Images/ImageService.cs

```diff
@@ -440,9 +449,9 @@ public Task Post(PostUserImage request)
         /// <param name="request">The request.</param>
         public Task Post(PostItemImage request)
         {
-            var id = GetPathValue(1);
+            var id = Guid.Parse(GetPathValue(1));
 
-            request.Type = (ImageType)Enum.Parse(typeof(ImageType), GetPathValue(3), true);
+            request.Type = Enum.Parse<ImageType>(GetPathValue(3).ToString(), true);
```

Comment:
I don't like those hardcoded numbers of where in query is the argument (like `3`), can this be handled more elegantly?

Reply:
Maybe, but that’s out of the scope of this PR

Reply:
that's true...